index.html

```diff
@@ -504,12 +501,12 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
             }
 
             // update features (limit 5)
-            var feature_content = "<h2>" + i18n.t("index.page.content.new-features.new-features-in", { defaultValue: "New in " + build.versionString, sprintNumber: build.versionString }) + "</h2>";
-            build.newFeatures.slice(0, Math.min(build.newFeatures.length, 5)).forEach(function (feature) {
-                feature_content += "<h3>" + $("<div/>").text(feature.name).html() + "</h3>";
-                feature_content += "<p>" + $("<div/>").text(feature.description).html() + "</p>";
+            var feature_content = '<h2>' + i18n.t("index.page.content.new-features.new-features-in", { defaultValue: "New in " + build.versionString, versionString: build.versionString }) + '</h2>';
+            build.newFeatures.slice(0, 5).forEach(function (feature) {
+                feature_content += '<h3>' + $("<div/>").text(feature.name).html() + '</h3>';
+                feature_content += '<p>' + $("<div/>").text(feature.description).html() + '</p>';
```

Comment:
I tried to streamline quotes a little, so every HTML literal uses `'` as outer quotes.